### DISCUSSION TITLE
path due to this access issue

### DISCUSSION THREAD

**Discussion by vasreshetov** [semaphoreui/semaphore#1176] [General]
I use docker image 2.8.74
My playbook run in cli "mode" ansible without problem, but in gui result failed:

Skipped '/home/user1/' path due to this access issue: [Errno 13]
Access denied /home/user1/

P.S.   become: true     I used

**Comment by ansibleguy** [comment]
That is VERY few information to work with.
Please supply:
* How did you install Ansible Semaphore (docker,deb,rpm)
* If docker
  * I would re-install the docker image as described [in the documentation](https://docs.ansible-semaphore.com/administration-guide/installation#docker)
* If deb/rpm
  * Make sure you have followed the [installation documentation](https://docs.ansible-semaphore.com/administration-guide/installation#package-manager)
  * Provide the content of the systemd-file to run the service (if any)
  * Is Semaphore run as root or another user?
  * Provide the 'config.json' content you are using
